keras_retinanet/preprocessing/csv_generator.py

```diff
@@ -0,0 +1,117 @@
+"""
+Copyright 2017-2018 Fizyr (https://fizyr.com)
```

Comment:
Thanks for the contribution, I really like the simplicity and re-usability of this generator.

We can't claim copyright for this though. In general you can not give away copyright without signing a legal document. So you should put your own copyright notice here.

If you developed this as part of your job (as employee) you should credit your employer. Otherwise, you can credit yourself (or a pseudonym if you don't want personal details in the file).